environments/kolla/configuration.yml

```diff
@@ -60,10 +60,6 @@ grafana_enable_unified_alerting: "yes"
 
 enable_mariabackup: "yes"
 
-# rabbitmq
-
-rabbitmq_server_additional_erl_args: "+S 1:1"
```

Comment:
The default is "+S 2:2 +sbwt none +sbwtdcpu none +sbwtdio none", which uses 2 Cores instead of one, maybe we want to modify that to stick with one core. But we can do some testing with that first.